### PR TITLE
Remove empty plain/JavaDoc comments

### PR DESCRIPTION
**Pull request by thc202**
Remove empty plain/JavaDoc comments (for example, `/* */` and variations
of `/** */`).
Also, in ContextListTableModel remove commented logger declaration.

### CODE REVIEW

File: src/org/zaproxy/zap/view/ContextListTableModel.java [hunk 1]
```diff
@@ -40,9 +40,6 @@
     
     //private static Logger log = Logger.getLogger(ContextListTableModel.class);
```
Comment (kingthorin) [inline src/org/zaproxy/zap/view/ContextListTableModel.java]: Not specifically related,  but could go too? (With a minor tweak to the commit msg.)

**Comment by kingthorin** [comment]
Looks good

File: src/org/zaproxy/zap/view/ContextListTableModel.java (hunk 1 above)
Comment (thc202) [inline src/org/zaproxy/zap/view/ContextListTableModel.java]: Done, updated the commit message and pull request.

**Comment by kingthorin** [comment]
Looks goodLeave a comment

**Pull request merged**